packages/next/src/server/config.ts

```diff
@@ -428,6 +428,16 @@ function assignDefaults(
     )
   }
 
+  if (result.outputFileTracing === false) {
```

Comment:
Do we also want to add a `@deprecated` to the config type?

Reply:
added in [`c88f3a0` (#57467)](https://github.com/vercel/next.js/pull/57467/commits/c88f3a0a826a9edde3e4b9b0085abf41b53a9a91)